layrry-cli/src/main/java/org/moditect/layrry/cli/LayrryRunner.java

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Copyright 2020 The ModiTect authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.moditect.layrry.cli;
+
+import com.beust.jcommander.JCommander;
+import org.moditect.layrry.Layrry;
+import org.moditect.layrry.cli.internal.Args;
+
+import java.io.File;
+
+/**
+ * The main entry point for using Layrry on the command line. Expects the layers config file to be passed in:
+ * <p>
+ * <code>
+ * Layrry --layers-config &lt;path/to/layrry.yml&gt;
+ * </code>
+ */
+public final class LayrryRunner {
+
+    public static void main(String... args) throws Exception {
+        run(args);
+    }
+
+    public static void run(String... args) throws Exception {
```

Comment:
Any strong reason for not doing this in `main()`?

Reply:
Not really, no. It just feels "odd" to invoke `main` directly.